src/style.css

```diff
@@ -1,3 +1,76 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+}
+
+:root {
+  --primary-color: #00acdc;
+}
+
+header {
+  justify-content: space-between;
+  display: flex;
+  background: #fff;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.ul {
+  display: flex;
+  align-items: stretch;
+  height: 60px;
+  list-style: none;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav-li {
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. Instead of `.nav-li`, use a more semantic class name like `.nav__item`.